Storage/SQL/SQLiteHistory.swift

```diff
@@ -404,221 +579,48 @@ extension SQLiteHistory: BrowserHistory {
         }
 
         let sql = [
-        "SELECT",
+            "SELECT",
```

Comment:
I don't mind either style of indentation here, but I'd prefer this patch to not change whitespace unnecessarily.